src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/methodname/Example7.java

```diff
@@ -0,0 +1,21 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodName">
+       <property name="format" value="^[a-z](_?[a-zA-Z0-9]+)*$"/>
+       <property name="applyToPackage" value="false"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.methodname;
+
+// xdoc section -- start
+class Example7 {
+  public void Method1() {} // violation 'Name 'Method1' must match pattern'
+  protected void Method2() {} // violation 'Name 'Method2' must match pattern'
+  private void Method3() {} // violation 'Name 'Method3' must match pattern'
+  void Method4() {} // ok, package-private methods ignored
```

Comment:
please remove "ok" comment, no comment means all is ok.

same for other Example.